### ISSUE TITLE
Inconsistent Module Naming Between Android and React Native

### ISSUE THREAD

**Issue by RicardoGuiliani** [tectiv3/react-native-aes#92]
Hi @tectiv3, thank you for this useful package! 

I've encountered a small issue related to module naming that affects the integration between the React Native and Android sides.

Problem:
In the Android Java module, the getName() method returns "RCTAes", as seen here:
```java
public class RCTAes extends ReactContextBaseJavaModule {
    ...
    
    @Override
    public String getName() {
        return "RCTAes";
    }
    
    ...
}
```

However, on the React Native side, the module is imported and used as "Aes":
```js
import { NativeModules } from 'react-native';

export default NativeModules.Aes;
```

This discrepancy between "Aes" and "RCTAes" makes the Android module inaccessible on React Native.

Suggested Solution:
Align the module names by either:

1. `Updating getName() in the Android module to return "Aes", or`
2. `Changing the React Native import to match the "RCTAes" name used in the Android module.`

Thank you.

**Comment by tectiv3** [comment]
Hi!
What do you mean by "inaccessible on React Native"? 
This module works on both ios and android.

**Comment by RicardoGuiliani** [comment]
Hi @tectiv3!

Thank you for your prompt response.

Currently, this module doesn’t work on Android due to the naming inconsistency mentioned above. Please double-check: if the module name in the Java code is "RCTAes" but is bridged in React Native as "Aes" (using NativeModules.Aes), it won't work as expected. I was able to make it work by importing it directly as NativeModules.RCTAes instead.

Could you please double-check this?

Thanks again!

**Comment by tectiv3** [comment]
yeah, it works and always worked. I don't think that discrepancy matters

**Comment by RicardoGuiliani** [comment]
> yeah, it works and always worked. I don't think that discrepancy matters.
> 
> That demo project (https://github.com/tectiv3/react-native-aes-demo) doesn't work for you either?

No, it doesn't work on android. 

It seems to be a common problem based on the other reports in the repo. 

I was able to resolve it by importing the module directly, which I noticed other users have done as well and some have even forked the repo to implement their own fixes.

Anyways, I just wanted to bring this to your attention in case you'd like to address it.

Thank you!
